1999/0xxx/CVE-1999-0006.json

```diff
@@ -34,7 +34,7 @@
       "description_data" : [
```

Comment:
{
   "CVE_data_meta" : {
      "ASSIGNER" : "cve@mitre.org",
      "ID" : "CVE-1999-0006",
      "STATE" : "PUBLIC"
   },
   "affects" : {
      "vendor" : {
         "vendor_data" : [
            {
               "product" : {
                  "product_data" : [
                     {
                        "product_name" : "n/a",
                        "version" : {
                           "version_data" : [
                              {
                                 "version_value" : "n/a"
                              }
                           ]
                        }
                     }
                  ]
               },
               "vendor_name" : "n/a"
            }
         ]
      }
   },
   "data_format" : "MITRE",
   "data_type" : "CVE",
   "data_version" : "4.0",
   "description" : {
      "description_data" : [
         {
            "lang" : "eng",
            "value" : "This is a test."
         }
      ]
   },
   "problemtype" : {
      "problemtype_data" : [
         {
            "description" : [
               {
                  "lang" : "eng",
                  "value" : "n/a"
               }
            ]
         }
      ]
   },
   "references" : {
      "reference_data" : [
         {
            "name" : "19980801-01-I",
            "refsource" : "SGI",
            "url" : "ftp://patches.sgi.com/support/free/security/advisories/19980801-01-I"
         },
         {
            "name" : "133",
            "refsource" : "BID",
            "url" : "http://www.securityfocus.com/bid/133"
         }
      ]
   }
}